source/index.md

```diff
@@ -600,9 +602,46 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
-##### Method: `mobilepayonline`
 
-<dl class="dl-horizontal">
+---
+##### **Method**: `googlepay`
+
+To accept a payment using Google Pay, apart from the complete payment method
+token and merchant ID, the derived shared secret is required. This must be
+derived according to the documentation.
+
+The cryptography involved is described [here][GooglePay-PaymentCryptography].
+
+<dl>
+  <dt>googlepay[token]</dt>
+  <dd>[:json:] <br /> Raw payment method token as received in response from Google. UTF-8
+    encoded serialization of a JSON dictionary.</dd>
+  <dt>googlepay[shared_secret]</dt>
+  <dd>
```

Comment:
```suggestion
  <dd>[:hex:]
```
?

Reply:
No, I haven't been able to find any format information for `merchantID`.